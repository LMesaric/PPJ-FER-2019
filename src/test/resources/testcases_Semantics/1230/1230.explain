semantic: invalid escaped string '\w' on byte offset 1
note: '"\w"' on line 2
