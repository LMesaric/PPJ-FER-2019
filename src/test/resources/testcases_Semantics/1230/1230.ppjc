int main(void){
char c[123]="\w";
return 0;
}