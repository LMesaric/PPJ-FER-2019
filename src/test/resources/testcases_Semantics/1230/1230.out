<primarni_izraz> ::= NIZ_ZNAKOVA(2,"\w")
